packages/devextreme/testing/tests/DevExpress.ui.widgets/listParts/commonTests.js

```diff
@@ -320,18 +320,6 @@ QUnit.module('collapsible groups', moduleSetup, () => {
         themes.isMaterialBased = origIsMaterialBased;
     });
 
-    QUnit.test('no group header collapsed indicator element for the Generic theme', function(assert) {
```

Comment:
There is a test with name 'group header collapsed indicator element for the Material theme' where we may remove isMaterial theme forcing and test it for common generic theme with thought that there is not some specificity through themes.